cmd/multitenant-autoscaler/main.go

```diff
@@ -79,12 +84,21 @@ func main() {
 		logger.Fatal("Error building kubernetes clientset.", zap.Error(err))
 	}
 
+	// This is based on how Kubernetes sets up its scale client based on discovery:
+	// https://github.com/kubernetes/kubernetes/blob/94c2c6c84/cmd/kube-controller-manager/app/autoscaling.go#L75-L81
```

Comment:
How about putting these kinds of comments in the commit log as they'll tend to go stale here as the code mutates?